lms/templates/dashboard.html

```diff
@@ -314,6 +316,29 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
       </div>
     </div>
 </main>
+%if show_account_activation_popup:
+    <div id="activate-account-modal" class="modal activate-account-modal" aria-hidden="true">
+      <div class="inner-wrapper" role="dialog" aria-labelledby="activate-account-modal-title" aria-live="polite">
+          <h3>
+             ${_("Activate your account so you can log back in")}
+            <span class="sr">,
+              ## Translators: this text gives status on if the modal interface (a menu or piece of UI that takes the full focus of the screen) is open or not
+              ${_("window open")}
+            </span>
+        </h3>
+        <form id="activate-account-form" style="padding: 0 1rem" method="post" data-remote="true" action="send-email">
+          <p>${Text(_("We sent an email to {email} with a link to activate your account. Can’t find it? Check your spam folder or {link_start}resend the email.{link_end}")).format(
+              email=user.email,
+              link_start=HTML('<a href="send_account_activation_email" id="send_cta_email" >'),
+                  link_end=HTML('</a>')
+              )}
+          </p>
+        </form>
+           <div style="text-align: right; display: block; padding: 0 0 1rem 0"><button id="button"> ${Text(_("continue to {platform_name}")).format(platform_name=settings.PLATFORM_NAME)}</button></div>
```

Comment:
Set `class="btn btn-primary"` on button. Button is also not aligned with the text from right side.